test/roundtrip/invalid-name.txt

```diff
@@ -0,0 +1,13 @@
+;;; TOOL: run-roundtrip
+;;; ARGS: --stdout --no-check --enable-annotations --debug-names
```

Comment:
Why is `--no-check` used here -- what makes the module invalid?

Reply:
Ah, I think I must've left this in when looking at another roundtrip test for reference. Thanks for the catch!